### ISSUE TITLE
Missing countries in the list

### ISSUE THREAD

**Issue by AdrianB1** [blazeworx/flagstrap#13]
Comparing to ISO list of countries, there are 7 missing countries:
Åland Islands, Antarctica, Bonaire, Sint Eustatius and Saba, Christmas Island, Cocos (Keeling) Islands (the), Saint Barthélemy, Svalbard and Jan Mayen. I do not comment on how real countries as these, but they have ISO country codes (AX, AQ, BQ,CX, CC, BL and SJ) and flags.

**Comment by blazeworx** [comment]
I don't have a problem adding AX, AQ, CX, CC, SJ and BL. They are codes for an individual country / territory and they each have one flag. SJ is its own country code but flies the flag of Norway so that is not a problem. However the problem is BQ. It is actually 3 special municipalities of the Netherlands with one code but 3 flags. 

![image](https://cloud.githubusercontent.com/assets/4718238/12520254/7ac8e702-c111-11e5-9cbb-a9647d52d637.png)

In this case I will have to use their BQ-BO code maybe. I will have to think about this some more. Moving this to a v1.2 miletone for the time being.

**Comment by AdrianB1** [comment]
Right, they have all individual flags but there is also this: https://en.wikipedia.org/wiki/Caribbean_Netherlands that is suggesting they are using the Nederlands flag for the BQ group and individual flags if taken separately. I would put the Nederlands flag, BQ code and "Bonaire, Sint Eustatius and Saba" as name.

**Comment by blazeworx** [comment]
:+1: I agree. I will make the changes.

**Comment by AdrianB1** [comment]
Any idea when you will add these countries? If there is any way I can help, I will try, but based on what I saw in the code it should be a 15 minute change.

**Comment by telemoes** [comment]
Sorry but the thee BQ's are cities of the Netherlands. It are not countries. Every city in the Netherlands have the own flag search on google flag Amsterdam or Rotterdam
